browser/ui/views/tabs/brave_tab_group_header.h

```diff
@@ -8,17 +8,30 @@
 
 #include "chrome/browser/ui/views/tabs/tab_group_header.h"
 
+namespace tab_groups {
+class TabGroupId;
+}  // namespace tab_groups
+
 class BraveTabGroupHeader : public TabGroupHeader {
  public:
   METADATA_HEADER(BraveTabGroupHeader);
 
-  static int GetLeftPaddingForVerticalTabs();
+  constexpr static int kPaddingForGroup = 4;
+
+  static SkColor GetDarkerColorForGroup(const tab_groups::TabGroupId& group_id,
```

Comment:
isn't this just the color for vertical tabs? `GetDarkerColor` isn't really a great name imo